src/store/modules/order/actions.ts

```diff
@@ -35,13 +35,44 @@ const actions: ActionTree<OrderState , RootState> ={
       }
       emitter.emit("dismissLoader");
     } catch(err) {
-      console.log(err)
+      console.error(err)
       showToast(translate("Something went wrong"))
     }
 
     return resp;
   },
 
+  async getOrderDetail( { dispatch, state }, payload ) {
+    const current = state.current as any
+    const orders = state.open.list as any
+
+    if(current.orderId === payload.orderId) { return current }
+    if(orders.length) {
+      return orders.some((order: any) => {
+        if(order.orderId === payload.orderId) {
+          dispatch('updateCurrent', { order })
+          return current;
+        }
+      })
+    }
+    
+    let resp;
+    try {
+      resp = await OrderService.getOpenOrder(payload)
+      if (resp.status === 200 && resp.data.count > 0 && !hasError(resp)) {
+        const order = resp.data.docs
+
+        this.dispatch('product/getProductInformation', { orders: order })
+        dispatch('updateCurrent', { order: resp.data.docs[0] })
+      } else {
+        showToast(translate("Order not found"))
```

Comment:
Please add it to en.json

Reply:
Added this missing label on en.json file.